nuget/lib/dependabot/nuget/file_parser/project_file_parser.rb

```diff
@@ -43,6 +43,18 @@ def dependency_set(project_file:)
             dependency = build_dependency(name, req, version, prop_name, project_file)
             dependency_set << dependency if dependency
           end
+
+          # Look for regular package references (development)
+          doc.css(DEV_SELECTOR).each do |dependency_node|
+            name = dependency_name(dependency_node, project_file)
+            req = dependency_requirement(dependency_node, project_file)
+            version = dependency_version(dependency_node, project_file)
+            prop_name = req_property_name(dependency_node)
+
+            dependency = build_dependency(name, req, version, prop_name, project_file, true)
+            dependency_set << dependency if dependency
+          end
```

Comment:
Could this be folded into the existing dependency selector and checking the `dependency_node` to see if it's a dev dependency?

Reply:
Yep, I just pushed that change for your review.